src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/RuntimeImports.cs

```diff
@@ -1262,8 +1262,9 @@ ushort ClrInstanceID
         internal static unsafe partial void* memset(byte* mem, int value, nuint size);
 
 #if TARGET_X86 || TARGET_AMD64
-        [LibraryImport(RuntimeLibrary)]
-        internal static unsafe partial void RhCpuIdEx(int* cpuInfo, int functionId, int subFunctionId);
+        [MethodImplAttribute(MethodImplOptions.InternalCall)]
```

Comment:
This should be fixed to be QCALL on the unmanaged side instead. (Same as regular CoreCLR.)

Reply:
Done